examples/tests/lr1110/wifi_scan/README.md

```diff
@@ -0,0 +1,179 @@
+that will be called when the Wi-Fi middleware sends an event.
+
+In this callback function, the application will process the pending events, and
+for any event except the `WIFI_MW_EVENT_SCAN_DONE` event, it will program the
+next scan with the delay defined by `WIFI_SCAN_GROUP_PERIOD`.
+
+## Configuration
+
+When compiling with *arm-none-eabi-gcc* toolchain, all these constant are configurable through command line with the `EXTRAFLAGS`.
+See [README.md](../../../README.md#command-line-configuration).
+
+### LoRaWAN related configuration
+
+The `apps/common/lorawan_key_config.h` header file defines several constants to
+configure the LoRaWAN parameters (profile, region, keys).
+
+| Constant         | Comments                         | Possible values      | Default Value              |
+| ---------------- | -------------------------------- | -------------------- | -------------------------- |
+| `LORAWAN_REGION` | Selects the regulatory region    | See here-below       | `SMTC_MODEM_REGION_EU_868` |
+| `LORAWAN_CLASS`  | Selects the LoRaWAN class to use | `SMTC_MODEM_CLASS_A` | `SMTC_MODEM_CLASS_A`       |
+
+Supported values for `LORAWAN_REGION`:
+
+* `SMTC_MODEM_REGION_EU_868 (default)`
+* `SMTC_MODEM_REGION_US_915`
+* `SMTC_MODEM_REGION_CN_470_RP_1_0`
+
+Supported values for `LORAWAN_CLASS`:
+
+* `SMTC_MODEM_CLASS_A`
+
+### Wi-Fi demonstration related configuration
+
+The `main_geolocation_wifi.h` header file defines several constants which can be
+set to define the configurable parameters of the application.
+
+| Constant           | Comments                                                                                      | Possible values | Default Value |
+| ------------------ | --------------------------------------------------------------------------------------------- | --------------- | ------------- |
+| `WIFI_SCAN_PERIOD` | Defines the duration between the end of a scan & send sequence and the start of next sequence | `uint32_t`      | 30            |
+## Build
+
+The demo can be built through GNU make command by doing the following:
+
+```shell
+# Navigate to the build folder
+$ cd apps/geolocation_wifi/makefile
+
+# Execute the make call
+$ make -j
+```
+
+By default, the demonstration is compiled to use the EUIs and Application key
+defined in the file *apps/common/lorawan_key_config.h*.
+
+## Usage
+
+### Serial console
+
+The application requires no user intervention after the static configuration
+option have been set.
+
+Information messages are displayed on the serial console, starting with the
+DevEUI, AppEUI/JoinEUI and PIN that you might need in order to register your
+device with the LoRa Cloud Device Join service.
+
+### LoRaWAN Network Server / Application Server
+
+This application needs an Application Server to run in order to perform the Wi-Fi
+solving.
+Two possibilities :
+* A Node-Red application server is provided in folder *apps/geolocation_application_server*.
```

Comment:
Is it?

Reply:
No. This file is copied from [seeed's repo](https://github.com/ethanzhouyc/Seeed_Wio_WM1110_Dev_Board/blob/master/apps/examples/geolocation_wifi/README.md) just as other files, and we should write our own version

Reply:
I suggest deleting this file and add a link to the original README on top of the file

Reply:
Sounds good